MapboxDirections/MBRouteOptions.swift

```diff
@@ -100,18 +100,24 @@ public enum RouteShapeResolution: UInt, CustomStringConvertible {
     }
 }
 
-@objc(MBVoiceUnit)
-public enum VoiceUnit: UInt, CustomStringConvertible {
+/**
+ A `MeasurementSystem` indicates the type of units used for formatted a voice instruction.
```

Comment:
Some of this detail is better left to the `RouteOptions.distanceMeasurementSystem` documentation. How about:

> A system of units of measuring distances and other quantities.